src/d3a/models/strategy/external_strategies/pv.py

```diff
@@ -374,52 +372,76 @@ def __init__(
                          update_interval=update_interval,
                          energy_rate_decrease_per_update=energy_rate_decrease_per_update,
                          use_market_maker_rate=use_market_maker_rate)
-        self.energy_forecast_buffer_Wh = energy_forecast_Wh
+
+        # Buffers for energy forecast and measurement values,
+        # that have been sent by the d3a-api-client in the duration of one market slot
+        self.energy_forecast_buffer = None  # Dict[DateTime, float]
+        self.energy_measurement_buffer = None  # Dict[DateTime, float]
```

Comment:
Since in type hint it is expected to be a dict with key as DateTime and a float value, why not initialising it with an empty dict?